### PR TITLE
Mention other reviews?

### PR DESCRIPTION
**Pull request by dziakj1**
I wanted to suggest citing some other review articles, but also noting the limitations of reviews at any static time.

### Suggested reviewers (optional)

@rando2 

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [x] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by dziakj1** [comment]
Another possible wording, to remove the negative-sounding word "dated," might be "it may also be beneficial to have an overall review which can evolve over time" instead of "any static review is likely to quickly become dated."

**Comment by dziakj1** [comment]
Or I guess you could just say "a more comprehensive review" instead of "a review which can evolve over time," since the latter might make people worry that your review will never be finished or never out of "alpha."

File: content/02.introduction.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ At this time, over 130,000 deaths had been reported due to COVID-19 worldwide (A
 [Note: Maybe add a graph here, update as new reports come out.]
 [So this is where manubot is particularly useful. it could pull from a public database / data table.]
 
+Some review articles on aspects of COVID-19 have already been published and may be of great usefulness.
+These have included reviews on epidemiological aspects (@doi:/10.1016/j.molmed.2020.02.008), immunological aspects (@doi:/10.1016/j.immuni.2020.05.002), and pharmacological treatments (@doi:/10.1016/j.immuni.2020.05.002).
```
Comment (rando2) [inline content/02.introduction.md:19]:
```suggestion
These have included reviews on the disease epidemiology [@doi:/10.1016/j.molmed.2020.02.008], immunological response [@doi:/10.1016/j.immuni.2020.05.002], and pharmacological treatments [@doi:/10.1016/j.immuni.2020.05.002].
```
The word "aspect" is used a few times, what do you think of this?

File: content/02.introduction.md [hunk 2]
```diff
@@ -15,6 +15,10 @@ At this time, over 130,000 deaths had been reported due to COVID-19 worldwide (A
...
+However, research on these topics proceed so quickly that any static review is likely to quickly become dated.
```
Comment (rando2) [inline content/02.introduction.md:20]:
```suggestion
However, research on these topics is proceeding so quickly that any static review is likely to quickly become dated.
```

File: content/02.introduction.md [hunk 3]
```diff
@@ -15,6 +15,10 @@ At this time, over 130,000 deaths had been reported due to COVID-19 worldwide (A
 [Note: Maybe add a graph here, update as new reports come out.]
 [So this is where manubot is particularly useful. it could pull from a public database / data table.]
 
+Some review articles on aspects of COVID-19 have already been published and may be of great usefulness.
```
Comment (rando2) [inline content/02.introduction.md:18]:
```suggestion
Some review articles on aspects of COVID-19 have already been published and may be very useful for contextualizing the disease within different disciplines.
```

**Comment by rando2** [comment]
> Or I guess you could just say "a more comprehensive review" instead of "a review which can evolve over time," since the latter might make people worry that your review will never be finished or never out of "alpha."

It's up to you! I don't think "dated" is too harsh but happy to go with whatever you think is best! (re: this comment and the one before)

**Comment by rando2** [comment]
@dziakj1 just wanted to ping you and see where you're at with this text! I had a few minor suggestions but it looks good overall!

**Comment by dziakj1** [comment]
Oh yes, it is fine!  I am very sorry -- I didn't know I had to approve.

**Comment by cgreene** [comment]
I'm guessing closing this was a mistake - reopening

**Comment by cgreene** [comment]
Based on @dziakj1's response to @rando2's suggestions - I have committed the suggestions and merged to the current master to get this ready to merge.

**Pull request merged**